backend/osmosewebsite/management/commands/seed.py

```diff
@@ -106,6 +109,8 @@ def _create_news(self):
         print(" ###### _create_news ######")
         for _ in range(0, random.randint(5, 15)):
             news = News.objects.create(
+                firstname=fake.first_name(),
+                lastname=fake.last_name(),
```

Comment:
Did you test it? News doesn't have firstname or lastname!

Reply:
I didn't touch the news, that was before